chia/wallet/puzzles/custody/member_puzzles/member_puzzles.py

```diff
@@ -152,3 +157,18 @@ def solve(self, secp_sk: EllipticCurvePrivateKey, message: bytes, my_puzzle_hash
             s = _s
         sig = r.to_bytes(32, byteorder="big") + s.to_bytes(32, byteorder="big")
         return Program.to([my_puzzle_hash, sig])
+
+
+@dataclass(frozen=True)
+class SingletonMember(Puzzle):
+    singleton_id: bytes32
```

Comment:
You should add some parametrizations for the singleton mod and launcher hash as well.  They can have defaults of the variables they used, but they should be customizable.